app/models/music.rb

```diff
@@ -0,0 +1,37 @@
+require 'rspotify'
+
+class Music
+  attr_reader :name, :url, :type, :image, :uri
+
+  def initialize(data)
+    @name = data.name
+    @url = data.external_urls["spotify"]
+    @type = data.type
+    @image = image_from_data(data)
+    @uri = data.uri
+  end
+  
+  def self.find(suggestion_array)
+    suggestion_array.map do |suggestion|
+      if suggestion["music_type"] == "artist"
+        data = RSpotify::Artist.find(suggestion["music_id"])
+      elsif suggestion["music_type"] == "album"
+        data = RSpotify::Album.find(suggestion["music_id"])
+      elsif suggestion["music_type"] == "track"
+        data = RSpotify::Track.find(suggestion["music_id"])
+      end
+      
```

Comment:
Good job writing this self method! 
It seems like a classic if-else code that maybe you could considered using "case" instead, it will make it a bit shorter and easier to read.